packages/api/.env.testing

```diff
@@ -1,4 +1,3 @@
 PORT=3334
 DB_DATABASE=sabia-testing
-NODE_ENV=testing
-APP_KEY=rgumfiszDWm3AgkKDXSTIZ6LzMfvMEv6
```

Comment:
essa é a key de escrita do algolia? se for, acho melhor remover o histórico de changes desse arquivo 💯

Reply:
é não, é uma key do adonis que ele usa pra gerar hashes e encriptação